src/test/java/Common.java

```diff
@@ -24,28 +24,11 @@
 
 /** Provide methods for testing. */
 public class Common {
-
   public static final boolean RECORD_MODE =
       System.getenv("AAS_CORE_AAS3_0_TESTS_RECORD_MODE") != null
           && System.getenv("AAS_CORE_AAS3_0_TESTS_RECORD_MODE").equalsIgnoreCase("true");
-  public static String TEST_DATA_DIR = Paths.get("test_data").toAbsolutePath().toString();
-
-  public static final List<String> CAUSES_XML_DESERIALIZATION_FAILURE =
-      Collections.unmodifiableList(
-          Arrays.asList(
-              "TypeViolation",
-              "RequiredViolation",
-              "EnumViolation",
-              "UnexpectedAdditionalProperty"));
 
-  public static final List<String> CAUSES_JSON_DESERIALIZATION_FAILURE =
-      Collections.unmodifiableList(
-          Arrays.asList(
-              "TypeViolation",
-              "RequiredViolation",
-              "EnumViolation",
-              "NullViolation",
-              "UnexpectedAdditionalProperty"));
+  public static String TEST_DATA_DIR = Paths.get("test_data").toAbsolutePath().toString();
 
   public static final List<String> CAUSES_FOR_VERIFICATION_FAILURE =
```

Comment:
I think you do not need this one anymore?

Reply:
Ack, well spotted.